src/host/Widget.ts

```diff
@@ -56,4 +61,9 @@ export class Widget {
         return this.messanger.on(type, handler);
     }
 
+    liveEventClosed() {
```

Comment:
doesn't the child need to know how to publish it too?

Reply:
added participantLeftMeeting messageType to child